src/Stache/Indexes/Index.php

```diff
@@ -136,6 +136,11 @@ public function forgetItem($key)
 
     abstract public function getItems();
 
+    public function getItemValue($item)
+    {
+        return (new \Statamic\Query\ResolveValue)($item, $this->name);
+    }
+
```

Comment:
Is this still needed? The rest of what was added to this class was removed in https://github.com/statamic/cms/pull/12894/commits/386f51acf8eb23e0b06c9b39ba60209486ec499b